app/views/layouts/application.html.erb

```diff
@@ -75,6 +75,11 @@
         </div>
       </div>
     </nav>
+    <% flash.each do |type, msg| %>
+      <div class='alert alert-success'>
+        <%= msg %>
+      </div>
+    <% end %>
```

Comment:
I had put my flash alert only on the page I wanted it to show on (i.e. not on the global template).  We'll need to double-check that this is not throwing duplicate alert messages after we merge.

Reply:
Moved to the two pages that use it. Thanks for the reminder!